utils/src/main/kotlin/storage/FileArchiver.kt

```diff
@@ -49,12 +49,17 @@ class FileArchiver(
 ) {
     companion object {
         private const val ARCHIVE_FILE_NAME = "archive.zip"
-        private val DEFAULT_ARCHIVE_DIR by lazy { ortDataDirectory.resolve("scanner/archive") }
+        internal val DEFAULT_ARCHIVE_DIR by lazy { ortDataDirectory.resolve("scanner/archive") }
```

Comment:
Could you please make it clearer in the commit message that this means, that the default `FileArchiver` does now match the license file patterns not only against the root directory, but also against every subdirectory, and why this enable reusing the archives?

Reply:
I updated the commit message, is it clear now?

Reply:
Yes, thanks.